orthoinference/README.md

```diff
@@ -2,29 +2,33 @@
 
 If the RlE passes all these tests, it is considered <i>eligible</i> for inference. Inference is first attempted on the RlE's <b>input</b> and <b>output</b> attributes, followed by <b>catalyst</b> and <b>regulation</b> inference attempts. <u>If the input or output inferences fail, then the process is terminated for that RlE since they are required components of any ReactionlikeEvent.</u> 
   
-During inference, the attribute (input/output/catalyst/regulation) is broken down into its individual <b>PhysicalEntity</b> (PE) instances. These PE instances are each run through the <a href="https://github.com/reactome/data-release-pipeline/blob/develop/orthoinference/src/main/java/org/reactome/orthoinference/OrthologousEntityGenerator.java">createOrthoEntity</a> method. This method infers according to PE type: <i>GenomeEncodedentity/EntityWithAccessionedSequence, Complex/Polymer, EntitySet or SimpleEntity</i>. In cases where the PE itself contains multiple <i>GKInstance</i> attributes (eg: Complexes <i>hasComponent</i> attribute, EntitySets <i>hasMember</i> attribute), these too are run through the createOrthoEntity method and inferred. Typically, the PE instance will be recursively broken down until they reach the <b>EntityWithAccessionedSequence</b> (EWAS) level and are inferred in the <a href="https://github.com/reactome/data-release-pipeline/blob/develop/orthoinference/src/main/java/org/reactome/orthoinference/EWASInferrer.java">inferEWAS</a> method.
+During inference, the attribute (input/output/catalyst/regulation) is broken down into its individual <b>PhysicalEntity</b> (PE) instances. These PE instances are each run through the <a href="https://github.com/reactome/data-release-pipeline/blob/develop/orthoinference/src/main/java/org/reactome/orthoinference/OrthologousEntityGenerator.java">createOrthoEntity</a> method. This method infers according to PE type: <i>GenomeEncodedentity/EntityWithAccessionedSequence, Complex/Polymer, EntitySet or SimpleEntity</i>. In cases where the PE itself contains multiple <i>GKInstance</i> attributes (eg: Complexes <i>hasComponent</i> attribute, EntitySets <i>hasMember</i> attribute), these too are run through the createOrthoEntity method and inferred. Through this system, PE instances will be recursively broken down until they reach the <b>EntityWithAccessionedSequence</b> (EWAS) level and are inferred in the <a href="https://github.com/reactome/data-release-pipeline/blob/develop/orthoinference/src/main/java/org/reactome/orthoinference/EWASInferrer.java">inferEWAS</a> method.
```

Comment:
For completeness, do you think Polymers should be mentioned for the PE types that contain multiple GKInstances?